editor/state.js

```diff
@@ -148,6 +148,15 @@ export function mode( state = 'visual', action ) {
 	return state;
 }
 
+export function sidebar( state = { opened: false }, action ) {
```

Comment:
I hesitated, I was expecting we may have a property saying we're showing the `PostSettings` or the `BlockInspector`, but yeah. I'll go for `isSidebarOpened` right now!